openedx_events/learning/data.py

```diff
@@ -217,3 +217,19 @@ class PersistentCourseGradeData:
     percent_grade = attr.ib(type=float)
     letter_grade = attr.ib(type=str)
     passed_timestamp = attr.ib(type=datetime)
+
+
+@attr.s(frozen=True)
+class XBlockSkillVerificationData:
+    """
+    Data needed to update verification count  of skill for an XBlock.
```

Comment:
can we add a bit of context of what a skill is? I'm not familiar with them

Reply:
Updated!